metatron-app/metatron-discovery
# #2562 Support realtime datasource

### Description

Support realtime datasource

**Related Issue** : #2562 



### How Has This Been Tested?

1. Create a datasource with stream
2. Input kafka server and choose topic
3. Check that the datasource is created and ingested normally.

#### Need additional checks?
https://metatron.app/2018/08/02/visualize-real-time-data-with-metatron-discovery/

### Types of changes



### Checklist:


### Additional Context
